src/app/components/MediaLoader/configs/audio.ts

```diff
@@ -0,0 +1,50 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: audioMediaBlock } = filterForBlockType(blocks, 'audio');
+  const { model: mediaOverrides } =
+    filterForBlockType(blocks, 'mediaOverrides') || {};
+  const audio = audioMediaBlock?.versions?.[0] || {};
+  const holdingImageURL = `https://${audioMediaBlock.imageUrl}`;
+
+  return {
```

Comment:
Do we need to include the following config to match the SMP config in Morph?
```
  responsive: true
  continuousPlay: false
  ui: {
    controls: {
      includeNextButton: false,
      includePreviousButton: false,
    },
    playbackSettingsPanel: {
      enabled: false,
      continuousPlaySetting: false,
    },
    cta: {
      mode: 'duration',
    },
  }
```
See [generateSMPSettings](https://github.com/bbc/morph-modules/blob/0f16bd34fad7339f0a61e5dcff3e489dbba9de3d/news-av-player/src/on-demand-audio/index.js#L25) for reference.

Reply:
Ah yes, we might. I think I assumed that these were the default settings, and if we did not set them in the settings, they would just get the default (=false) values. We were also comparing this to the settings for live radio, so we've taken much of the config from there. 

I'm not against adding this if we need to, but I wondered if we could/should wait until we put the player on the page, and see if these settings are still required?

Reply:
Sounds good.
Let's merge this and evaluate the settings
after the MediaLoader is added to the page.